docs/release-notes.md

```diff
@@ -12,6 +12,53 @@ Welcome to Semgrep release notes. This document provides an overview of the chan
 
 ## December 2021
 
+### Version 0.77.0
+
+#### Highlights
+
+##### Semgrep CLI and Semgrep CI now ignore the same patterns
+
+With this update, Semgrep CLI now ignores the same patterns as the Semgrep CI by default. Find [the default .semgrepignore on GitHub](https://github.com/returntocorp/semgrep/blob/develop/semgrep/semgrep/templates/.semgrepignore). If you want to return to Semgrep’s previous behavior, create an empty `.semgrepignore` file. However, creating a new `.semgrepignore` overrides the default setup.
+
+##### Autofix improvement
+
+An autofix improvement from https://github.com/chair6 from Hashicorp! Big shoutout to them. Fixes several issues (auto fixing multiple things in the same set of lines)
+
+#### Additions
+
+##### Scala
+
+Semgrep now correctly matches patterns as `List(...)`.
+
+##### `semgrepignore.`
+Default set of `.semgrepignore` patterns (in `semgrep/templates/.semgrepignore`) are now used by default. You can override the default behavior by your own setup of `semgrepignore.` file.
+
+##### Java
+
+You can now use ellipsis metavariables for parameters. ([#4420](https://github.com/returntocorp/semgrep/issues/4420))
+
+#### Fixes
+
+The fixed section now remains only as changelog notes. To see the changelog notes, visit [Semgrep changelog](https://github.com/returntocorp/semgrep/releases/tag/v0.77.0).
+
+#### Changes
+
+##### Constant propagation
+
+Constant propagation is now fully a must analysis, if a variable is undefined in some path then it is considered as a non-constant.
+
+##### Dataflow
+
+Dataflow now considers only reachable nodes, which prevents some false-positive or false-negative findings.
+
+##### The `–time` option now includes time spent on processing
```

Comment:
Is it `-time` or `--time`?

Reply:
`--time`! Will fix it. 

![Screenshot 2021-12-21 at 21 12 52](https://user-images.githubusercontent.com/26669307/146991867-450904d6-4831-49f7-9d3b-b6fe4588cd3c.jpg)

Reply:
Fixed.

Reply:
Resolving this comment! :-)